app/views/staff/shared/activities/_activity.html.haml

```diff
@@ -17,7 +17,7 @@
 
   .govuk-summary-list__row.title
     %dt.govuk-summary-list__key
-      = t("summary.label.activity.title")
+      = t("summary.label.activity.title", level: activity_presenter.level.humanize)
```

Comment:
Not sure if it matters, but https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/480/files#diff-f0cc68c5ea5e4239efd88b1881df8dddR4 calls `humanize` on the whole I18n string rather than the level part, i.e.

`= t("summary.label.activity.title", level: activity_present.level).humanize`

Reply:
Again, I think both work and it could probably use `capitalize`.